apps/golden-sample-app-e2e/fixtures/transaction-details.fixture.ts

```diff
@@ -0,0 +1,21 @@
+import { test as baseTest } from '../page-objects/test-runner';
+import {
+  TransactionDetailsFixture,
+  TransactionDetailsPage,
+} from '@backbase-gsa/transactions-journey/e2e-tests';
+import { transactionDetailMocksData, transactionDetailSandboxData } from '../data/transactions-detail.data';
+
+export const test = baseTest.extend<TransactionDetailsFixture>({
+  detailsPage: async ({ page, baseURL }, use) => {
```

Comment:
What's the reasoning behind passing baseURL here? I it because we can't guarantee what the `baseURL` will be for users of the shared library?